modulefiles/build.wcoss_cray

```diff
@@ -15,17 +15,26 @@ module load craype-haswell
 module load alps/5.2.4-2.0502.9822.32.1.ari
 module load cray-netcdf/4.3.3.1
 module load cray-hdf5/1.8.14
-module load w3nco-intel/2.0.6
-module load nemsio-intel/2.2.3
-module load bacio-intel/2.0.2
-module load sp-intel/2.0.2
-module load ip-intel/3.0.0
-module load sigio-intel/2.1.0
-module load sfcio-intel/1.0.0
-module load landsfcutil-intel/2.1.0
-module load gfsio-intel/1.1.0
-module load w3emc-intel/2.2.0
-module load nemsiogfs-intel/2.0.1
+
+module use /usrx/local/nceplibs/NCEPLIBS/cmake/install/NCEPLIBS_UFS/modules
```

Comment:
These were installed by you, correct?  Has a CDF been created for NCO?

Reply:
No. They were installed by the NCEPLIBS team.